### ISSUE TITLE
[venue] The first event needs a home

### ISSUE THREAD

**Issue by Qard** [dancejs/Vancouver#2]
Anyone have venue suggestions for the first event? I don't know what the size of the first event will be yet, but I want it to start with a bang!

Somewhere accessible would be great. Gastown, Yaletown or somewhere near Skytrain stops not too far from downtown would be best. The warehouse feel of Oakland event worked well, and it'd be nice to be able to bring some other cool art stuff in there. Oakland had dropship, Vancouver will have...?

**Comment by bcoe** [comment]
CC: @gsilk you said you might have some people you'd like to put @Qard in touch with?

**Comment by gsilk** [comment]
@bcoe I've contacted my buddy Ken Pratt via gmail, assuming he's interested I'll add him to this group?

**Comment by Qard** [comment]
@canuckistani mentioned http://openstudios.ca/.

**Comment by gsilk** [comment]
Here ya go -- @kenpratt

**Comment by kenpratt** [comment]
@gsilk and I attended an event at the hall above the Legion on Commercial Drive -- 2 blocks from the Skytrain. WISE hall is another common local place, but it's far from the Skytrain.

You could also get in touch with the Centre for Digital Media (http://thecdm.ca/). They have space, are close the VCC-Clark Skytrain, and might be into hosting this since it's a tech-y event.

HiVE hosts plenty of tech stuff, in Gastown, not sure about a dance party though: http://www.hivevancouver.com/

You may also want to ping the local VanJS and PolyglotVancouver meetup group organizers, they'd probably have suggestions for space and/or be willing to promote and maybe co-host.

**Comment by flyingoctopus** [comment]
the vancouver hackspace just found another space and will be moving in soon. I tend to host events there. They are a great community of people there.

**Comment by Qard** [comment]
I've already chatted a bit with @sintaxi about Node Brigade. Now that I'm back in town, I'll see about talking with @apike about VanJS.

I'm going to start properly evaluating venues this week and trying to figure out a date.

**Comment by gsilk** [comment]
@Qard you may also want to investigate The Anza (or The Anza Club, I forget which)

**Comment by flyingoctopus** [comment]
Bump for the hive.

**Comment by jeffgca** [comment]
@Qard @gsilk The Anza club is a good suggesstion - friends of mine hosted a 'producer's forum' event there last year associated with Sequential Circus and it was quite cheap for the daytime space - plus they have an existing sound system and an expectation from neighbours that things will get loud. If you want to do the event at night on a weekend you'd be paying regular rates, but that might be pretty cheap as well.

**Comment by Qard** [comment]
Yep, I'm shooting for a Saturday night to make it easy for Oakland folks to come up, if they want.

Anza looks interesting. The one thing I'm not entirely sure about is the SOCAN license stuff. Anyone have experience with that?

**Comment by affinity-matrix** [comment]
I would absolutely love to make it up for this! Do you have a date set yet? 

+1 for Gastown (the only part of Vancouver I'm familiar with).

**Comment by Qard** [comment]
Not yet. I'm trying for sometime in March though.

**Comment by Qard** [comment]
Does Saturday, May 16th work for any Oakland folks interested in coming up?

**Comment by bcoe** [comment]
@Qard May works for me. can't wait.

**Comment by affinity-matrix** [comment]
@Qard I've just been made to realize that Maker Faire is May 16th. That might make it tough for me! :frowning:

**Comment by Qard** [comment]
@emilyrose @bcoe @othiym23 

Boo.

Does May 23rd work? I haven't confirmed the venue reservation yet, so I may be able to move it to the next weekend. I have to confirm by Monday though.

Also, I'm trying to bring JSFest up here eventually, so I'd probably run another DanceJS with that too. No idea if/when that'll happen yet though. :)

**Comment by affinity-matrix** [comment]
@Qard I think it may be worth considering, I think the 23rd would work as well & may be less of a conflict for some of the folks we want to draw.

Count me in as far as future plans for a JSFest as well! :smile_cat:

**Comment by bcoe** [comment]
I'm happy with the 23rd too.

**Comment by othiym23** [comment]
23rd works for me, I'm pretty sure.

**Comment by lrlna** [comment]
Anza club is an awesome venue! They host quite a few shows on the weekends though, so might want to talk to them pretty soon! 

Also - their event calendar -- http://www.anzaclub.org/calendar/action~month/exact_date~1430509859/

**Comment by jeffgca** [comment]
I think Stephen has a date and venue?

**Comment by Qard** [comment]
In theory...if the venue ever gets back to me with the bill. It should be May 23rd.

**Comment by jeffgca** [comment]
Yeah, they're ...laid back.

**Comment by lrlna** [comment]
I probably should have seen that this issue got started two months ago.... anyway, if you guys need help organizing, I am here.

**Comment by affinity-matrix** [comment]
hey all! just wanting to confirm the date before I look at tickets & accommodations!

**Comment by flyingoctopus** [comment]
woah woah woah guys! where have I been? just now catching up on thread. what was the confirmed venue? I can help with many things related to this. and I would also like to cordially invite all y'allz to participate in the up coming vancouver makerfaire this June. I'm serious. Also: let me know if you want to do a music performance and/or workshop or table, I'll help expedite your applications. Here's the [link](http://makerfaire.ca)

**Comment by Qard** [comment]
It's May 23rd at Open Studios. @sintaxi is also working on a node event during the daytime before it, but I'm not sure what the status of that is yet.

**Comment by bcoe** [comment]
@Qard need any guest DJs, _cough_

**Comment by flyingoctopus** [comment]
@Qard open studios is a win. let's make sure we can fill the space. what are your avenues of distribution? should I post something about this on the make blog? is it ready for it?

**Comment by Qard** [comment]
@bcoe I certainly wouldn't protest to it! :)

@flyingoctopus I'm pairing it with the npm-themed node event @sintaxi is working on, so I guess it'd get promoted via the usual Node Brigade channels. I've been holding off on saying too much about it publicly until the details for the node event are sorted. Any other promotion would be welcome though. :)

**Comment by jeffgca** [comment]
@Qard depressing as it may seem initially, it might be useful to promote this event on Facebook ( FB seems to be the main way music events in Vancouver are promoted )

**Comment by Qard** [comment]
Looks like PolyglotConf just got announced for the same day. >.>

https://twitter.com/tavisrudd/status/571844516841299969

**Comment by affinity-matrix** [comment]
Sweet! let's be the afterparty?

**Comment by Qard** [comment]
We had planned on pairing this with Node School and Node Brigade in the morning and afternoon, but PolyglotConf just kind of stomped on that. :(

PolyglotConf has done their own after parties in the past, not sure what the timing is exactly, but I could possibly do DanceJS as the after-after party and see if @sintaxi can move the Node School and Node Brigade stuff to the 24th. I have the DanceJS venue until 4AM, so we can party hard. :)

**Comment by sintaxi** [comment]
I say we stick with what's planned. Vancouver can support both events.

**Comment by Qard** [comment]
I'm good either way. But the Node School and Node Brigade stuff overlaps completely, while the DanceJS stuff just catches the tail end.

Is it too late to shift the date for those forward a day? It'd be cool if people could attend both events. We could work together to do one big weekend, rather than stepping on each other...

**Comment by Qard** [comment]
Although May 23rd is Node School International Day, which would be super cool to synchronize with...

**Comment by othiym23** [comment]
I haven't bought plane tickets yet, so I'm flexible on scheduling.

**Comment by jeffgca** [comment]
I agree with @sintaxi I think Vancouver can handle a node event and Polyglot same-day. It's not perfect, there is overlap, but it's a pretty big scene these days.

**Comment by Qard** [comment]
Just so everyone is clear: May 23rd is the date. There'll be a Node School and an npm-themed thing in the afternoon before it, for anyone interested in coming out early.

I've got the venue for 8 hours, so if anyone wants to guest DJ or do a talk, let me know (or post an issue). :)

**Comment by sintaxi** [comment]
@Qard Mobify has offered their space for either the 23rd or the 24th. Are we committing to the 23rd for NodeBrigade and Node School?

**Comment by Qard** [comment]
Well, Node School at least probably wants to be on the 23rd for International Day. It seemed like you had already decided on 23rd for Node Brigade too, but it's up to you.
